doc/prototype/identity-keys.md

```diff
@@ -6,16 +6,17 @@
 
 @signature `queryLogic.identityKeys()`
 
-Return the identity keys used to identity instances associated with the query logic:
+  Return the identity keys used to identity instances associated with the query logic:
```

Comment:
We might want to deprecate this and show using `canReflect.getSchema( queryLogic ).identity` instead ...